### PR TITLE
Pip installable docs

### PR DESCRIPTION
**Pull request by jordis-ai2**
I also passed mypy in my setup (I needed to get rid of all `from torch import nn` and alike).

### CODE REVIEW

**Comment by Lucaweihs** [comment]
Can you give some details on why removing `from torch import nn` and `import torch.nn as nn` was necessary? Seems like a lot of maybe superfluous edits which can cause merge conflicts for others.

**Comment by jordis-ai2** [comment]
None of those two alternatives made mypy work in my setup (clean install).

**Comment by jordis-ai2** [comment]
After using `mypy --follow-imports=normal .` instead of `mypy .` I could revert back to `from torch import nn` and similar.

File: mkdocs.yml [hunk 1]
```diff
@@ -177,6 +177,7 @@ nav:
           - objectnav_robothor_rgb_resnetgru_dagger: api/projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgb_resnetgru_dagger.md
           - objectnav_robothor_rgbd_resnetgru_ddppo: api/projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgbd_resnetgru_ddppo.md
           - objectnav_robothor_rgb_resnetgru_ddppo_and_gbc: api/projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgb_resnetgru_ddppo_and_gbc.md
+          - objectnav_robothor_rgb_resnetgru_ddppo_wsgi: api/projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgb_resnetgru_ddppo_wsgi.md
```
Comment (Lucaweihs) [inline mkdocs.yml:180]: I don't think we need this?

**Review by Lucaweihs — COMMENTED** [review]
One minor comment but looks great! Feel free to merge.

**Pull request merged**